README.md

```diff
@@ -21,8 +24,8 @@ An incomplete (in a way not all packages are mentioned here) example of python c
 # Recipe for python collections
 ---
 python33:
```

Comment:
Yes, the `python33` is ID. ID means Identity document [1]. Unique string.
Technically it is same with the key of the hash map in the YAML.
The ID manages `name`, `packages` element (= 1 SCL)
1 recipe file manages a list of ID.

[1] https://en.wikipedia.org/wiki/ID